<ui version="4.0" >
 <class>SocketCfgDialog</class>
 <widget class="QDialog" name="SocketCfgDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configuración de Socket</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="toolTip" >
        <string>Dirección IP del destino.
Por defecto la placa MC WebDev tiene la direccion 192.168.1.8</string>
       </property>
       <property name="text" >
        <string>Dirección</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2" >
      <widget class="QLineEdit" name="lineAddr" >
       <property name="text" >
        <string>192.168.1.8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Puerto</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2" >
      <widget class="QSpinBox" name="spinPort" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>65536</number>
       </property>
       <property name="value" >
        <number>9761</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2" >
      <widget class="QLabel" name="label_3" >
       <property name="toolTip" >
        <string>Tiempo de espera en el establecmiento del socket</string>
       </property>
       <property name="text" >
        <string>Timeout</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QDoubleSpinBox" name="spinTimeOut" >
       <property name="suffix" >
        <string> segs.</string>
       </property>
       <property name="decimals" >
        <number>2</number>
       </property>
       <property name="minimum" >
        <double>0.010000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.100000000000000</double>
       </property>
       <property name="value" >
        <double>0.040000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SocketCfgDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>241</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>117</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SocketCfgDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>323</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel" >
     <x>273</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
